pkg/tracingpolicy/generictracingpolicy.go

```diff
@@ -198,15 +198,17 @@ func ValidateCRDSpec(policy K8sTracingPolicyObject) (*validate.Result, error) {
 				return
 			}
 			for _, ver := range internal.Spec.Versions {
-				var sv *validate.SchemaValidator
+				var sv validation.SchemaValidator
 				var err error
-				sv, _, err = validation.NewSchemaValidator(ver.Schema)
-				if err != nil {
-					validatorState.initError = err
-					return
+				if ver.Schema != nil {
+					sv, _, err = validation.NewSchemaValidator(ver.Schema.OpenAPIV3Schema)
+					if err != nil {
+						validatorState.initError = err
+						return
+					}
 				}
 				if internal.Spec.Validation != nil {
-					sv, _, err = validation.NewSchemaValidator(internal.Spec.Validation)
+					sv, _, err = validation.NewSchemaValidator(internal.Spec.Validation.OpenAPIV3Schema)
```

Comment:
did it worked? I got a panic last time when I tried but I guess I did something else wrong!

Reply:
i was just going to ask you to review it, i have not idea what i did 😂

Reply:
Haha I did exactly the same without having any clue, that's basically how I deal with the k8s codebase! It seems that the tests are okay, I need to check manually to make sure with an invalid policy that nothing explodes! I'll test when you click "Ready" 🟢 🥗

Reply:
alright it's ready ✅

Reply:
looks like it's working fine! should add some tests with invalid tracing policies I guess...